### DISCUSSION TITLE
How do the drop events work exactly?

### DISCUSSION THREAD

**Discussion by blackpurge** [TEHE-Studios/ExpandedHelicopterEvents#341] [Q&A]
Prefacing this with the fact that I am playing with a ton of mods so some might be interfering in some way, but my questions are as follows: 

1: Is a drop guaranteed if a FEMA or Good samaritan event spawns? 

2: If the answer to Q1 is no, then does using a flare make it more likely to drop? 

3: Do drops only occur if there is suitable terrain? do they occur only on vanilla map areas? I feel like drops don't happen on modded maps

4: If the answer to Q1 is yes, then are there any known mods to interfere with this mod? any other troubleshooting tips are welcome.

**Comment by Chuckleberry-Finn** [comment]
If you're using the radio to confirm it is a FEMA/Samaritan drop then yes - they'll always try to drop something. The issue is sometimes the game will decide an area is too tight to spawn the crates and refuse to spawn it. The mod will try to scan nearby areas to fit it but on occasion this can fail.

That being said, many people just don't see them, or they land in the middle of trees which obscure vision unless you walk right on it.

The mod uses the area around the player, and a heatmap of activity, so it will work on any map. Flares add a lot of activity to that heatmap which makes it more likely that events will target that area. It won't redirect events unless they're the type to redirect to new targets.

**Reply by blackpurge** [reply]
Thanks for the reply, Is it possible for the FEMA/Samaritan plane to drop something but the drop indicator does not appear? Also does the radio notify the type of drop? Aren't all propellor planes FEMA/Samaritan?

**Reply by Norcan21** [reply]
yesterday i was playing on my server with friends and this happened to me as well, the samaritan was announced on radio, plane flew above us but no indicator for the drop, nothing (we are all in the same base), here are the logs 
![image](https://github.com/user-attachments/assets/5b59e3d2-44fb-4cc7-9665-811c95ba38ce)
